test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -33,11 +33,15 @@ def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
         hide=True,
     )
 
-    ec2_connection.run(
+    test_output = ec2_connection.run(
         f"nvidia-docker exec --user root smdebug-gpu /bin/bash -c '{test_script} {framework}'",
-        hide=True,
+        hide=True, warn=True
     )
 
+    LOGGER.info(test_output.stdout)
+
+    assert test_output.ok, "smdebug tests failed"
```

Comment:
Can we add more info to the assertion error? perhaps logging stdout and stderr would be useful